.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: python
-  python:
-    - 3.6
-    - nightly
-  script:
-    - pytest
+python:
+  - "3.6"
+# command to install dependencies
+install:
+  - pip install .
+  - pip install -r requirements.txt
```

Comment:
Hope you have a requirements.txt file if not then find out what it is for and create it.